packages/cozy-client/src/hooks/useQueryAll.jsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect } from 'react'
+
+import { UseQueryReturnValue } from '../types'
+import { QueryDefinition } from '../queries/dsl'
+import useQuery from './useQuery'
+import useSafeState from './useSafeState'
+
+/**
+ * Fetches a queryDefinition and run fetchMore on the query until the query is fully loaded
+ *
+ * @param  {QueryDefinition} queryDefinition - Definition created with Q()
+ *
+ * @typedef {object} UseQueryOptions
+ * @property  {object} as - Name for the query [required]
```

Comment:
in JSDocs, every field added is required.

when optional, we should write `[enabled]` or `[enabled=true]` when we passed default value